packages/shared/styles/components/molecules/SfRadio.scss

```diff
@@ -88,16 +88,15 @@
       1.6,
       var(--font-family--primary)
     );
-    color: var(--radio-description-margin);
+    color: var(--c-text);
   }
   &.is-active {
     background: var(--radio-background, var(--c-light));
   }
   &.is-disabled {
     --radio-checkmark-border-color:  var(--c-text-disabled);
-    --radio-details-color: var(--c-text-disabled);
-    --radio-description-margin: var(--c-text-disabled);
-    --radio-label-color: var(--c-text-disabled);
+    --c-text: var(--c-text-disabled);
```

Comment:
If you want to override in this case colors in a disabled state (without using CSS vars) then you can do something like this:
```
&.is-disabled {
  .desired-class {
    color: <desired color>
  }
}
```
so if there are removed 3 CSS vars (`--radio-details-color`, `--radio-description-margin`, and `--radio-label-color`) you need to add `deatils`, ` description` and `label` classes that override CSS properties :)

Reply:
@justyna-13 yeah, to be honest I didn't really understand the desired approach while removing the component-specific variables. This started as a request to fix the bug where the `--radio-description-margin` variable uses a color value. But as you are removing these specific variables, it was requested that I submit the PR to fix this while following the new approach. I wasn't really sure how you intended to make this customizable while removing the variables. I think I understand a little better now.

As a user of SF UI, this change is a bit frustrating as we have to start either removing the `scoped` css, or using `::v-deep` everywhere to be able to customize the styles in these components, instead of just modifying specific css variables. We're going to have to rework a lot of our components we've been building.

Reply:
Ok, I made the requested updates. Is that closer to what you are aiming for?

Reply:
yes :) this is exactly what we aim for. I know that at first sight CSS variables were a great solution for customization but we received feedback that there is a problem with their specificity and sometimes caused more problems than solved. This is why we decided to remove them and use the old way. If you prefer to use CSS vars then you can copy all components to your project and stay with the version that uses them